src/Lockedfield.php

```diff
@@ -165,7 +175,20 @@ public function getLocks($itemtype, $items_id)
 
         $locks = [];
         foreach ($iterator as $row) {
-            $locks[] = $row['field'];
+            if ($fields_only === true) {
+                $locks[] = $row['field'];
+            } else {
+                $value = $row['value'];
+                //if it refers to a ForeignKey get dropdown name
+                if (
+                    $row['value'] !== null
+                    && isForeignKeyField($row['field'])
+                    && is_a($itemtype = getItemtypeForForeignKeyField($row['field']), CommonDropdown::class, true)
+                ) {
+                    $value = Dropdown::getDropdownName(getTableForItemType($itemtype), $row['value'], 0, true, true, $row['value']);
```

Comment:
The `$tooltip` arg should be `false`. 
```suggestion
                    $value = Dropdown::getDropdownName(getTableForItemType($itemtype), $row['value'], 0, true, false, $row['value']);
```